overlay/youtrack_rev.json.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+
+SPDX-License-Identifier: MPL-2.0
```

Comment:
Newline

Reply:
This is automatically generated by `reuse`, so manually adding a newline seems a little unusual

Reply:
Oh, ok, I'll stop complaining about `.license` files.